backend/src/services/user.service.ts

```diff
@@ -0,0 +1,44 @@
+import UserModel, { IUser } from '../models/User';
+import bcrypt from 'bcrypt';
+import User from '@interfaces/User';
+export const createUser = async (user: User): Promise<IUser> => {
+  try {
+    const salt = await bcrypt.genSalt(10);
```

Comment:
Don't use magic numbers.

```suggestion
import User from '@interfaces/User';
const SALT_ROUNDS = 10;
export const createUser = async (user: User): Promise<IUser> => {
  try {
    const salt = await bcrypt.genSalt(10);
```